pom.xml

```diff
@@ -89,9 +89,15 @@
             <version>5.7.2</version>
         </dependency>
         <dependency>
+<<<<<<< Updated upstream
             <groupId>nz.net.ultraq.thymeleaf</groupId>
             <artifactId>thymeleaf-layout-dialect</artifactId>
             <version>2.3.0</version>
+=======
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-search-orm</artifactId>
+            <version>5.11.1.Final</version>
+>>>>>>> Stashed changes
```

Comment:
Remove the merge lines from the commit

Reply:
This has been fixed a long time ago

Reply:
Go to this commit and remove those lines.

Reply:
should be fixed now.